src/Libraries/Watch3DNodeModelsWpf/Watch3DNodeViewCustomization.cs

```diff
@@ -46,9 +46,6 @@ public void CustomizeView(Watch3D model, NodeView nodeView)
             var dynamoViewModel = nodeView.ViewModel.DynamoViewModel;
             watch3dModel = model;
 
-            var renderingTier = (RenderCapability.Tier >> 16);
```

Comment:
interesting, why remove these, I think they are checking for shader model, texture size etc?

Reply:
This is unrelated, sort of. Just wanted to see if removing this causes any failures. The problem is, keeping this check prevents Watch3D nodes to work in software mode. It is not needed as far as I can tell.

Reply:
ahh, got it, it may make more sense to check device info directly or something like.

Reply:
I will revert this and move it to a separate PR.